examples/all-clusters-app/linux/main-common.cpp

```diff
@@ -61,6 +64,13 @@ NamedPipeCommands sChipNamedPipeCommands;
 AllClustersCommandDelegate sAllClustersCommandDelegate;
 chip::app::Clusters::WindowCovering::WindowCoveringManager sWindowCoveringManager;
 
+chip::app::Clusters::ResourceMonitoring::HepaFilterMonitoringInstance
+    gHepafilterInstance(0x1, chip::app::Clusters::HepaFilterMonitoring::Id, 3,
```

Comment:
```suggestion
    gHepafilterInstance(0x1, Clusters::HepaFilterMonitoring::Id, 3,
```

This file has `using namespace chip::app`.  Please get rid of all the redundant prefixes.

Reply:
done